app/uk/gov/hmrc/fileupload/testonly/TestOnlyController.scala

```diff
@@ -51,7 +52,7 @@ class TestOnlyController(fileRepo: FileRepository, envelopeRepo: EnvelopeReposit
   }
 
   private def cleanupEnvelopeAndFiles: Future[(WriteResult, List[WriteResult])] = {
-    envelopeRepo.removeAll() zip fileRepo.removeAll()
+    envelopeRepo.removeAll() zip fileRepo.clear(Duration.ZERO)
```

Comment:
Cheers, I agree. This cleanup change was not done on this branch though (already on master). Maybe make the change directly on master or another branch?